waku/v2/utils/logger.go

```diff
@@ -25,12 +25,12 @@ func SetLogLevel(level string) error {
 // Logger creates a zap.Logger with some reasonable defaults
 func Logger() *zap.Logger {
 	if log == nil {
-		InitLogger("console").Panic("Logger not yet initialized")
+		InitLogger("console")
```

Comment:
This is the most important change, we allow lazy init to "console" instead of panicing if InitLogger wasn't called yet.

Reply:
I'm alright with this, although we should at least log a warning in this case. It's going to be a little odd that the log encoding can potentially switch (console after some `init()` function calls for the `logger()`, and then `json` when the app starts up)

Reply:
OK, I'll change Panic to Warn.